app/models/oral_history_content.rb

```diff
@@ -108,6 +108,16 @@ def has_ohms_index?
   end
 
 
+  # This is required in order to use repeatable_attr_input .
+  # These methods aren't actually called.
+  def interviewee_school_attributes=(some_arr)
+  end
+  def interviewee_job_attributes=(some_arr)
+  end
+  def interviewee_honor_attributes=(some_arr)
+  end
```

Comment:
we shouldn't need these, not sure why we do -- compare to analagous embedded models in Work to figure out why it didn't need it there. Might be because of how the controller is handling assigning input.

Reply:
"If you have nested AttrJson::Models you'd like to use in your forms much like Rails associated records: Where you would use Rails accepts_nested_attributes_for, instead include AttrJson::NestedAttributes and use attr_json_accepts_nested_attributes_for. Multiple levels of nesting are supported."

This solved the problem and made the form much simpler.

Reply:
Excellent, nice job! This stuff is kind of confusing. I wonder if AttrJson should turn that on by default.